src/remixAppManager.js

```diff
@@ -79,7 +79,18 @@ export class RemixAppManager extends PluginEngine {
   async registeredPlugins () {
     const res = await fetch(this.pluginsDirectory)
     const plugins = await res.json()
-    return plugins.map(plugin => new IframePlugin(plugin))
+    const remixCodeExecutor = {
```

Comment:
Looks like opening Pandora box ^^. You cleaned up everything in the last PR to put it in IPFS, and now you're putting back a local profile of a plugin. 
If you want to create a plugin for real, then consider putting it in the remix-plugin-directory instead.

Reply:
ok good point